examples/src/main/scala/org/apache/spark/examples/HBaseTest.scala

```diff
@@ -36,7 +36,7 @@ object HBaseTest {
     // Initialize hBase table if necessary
     val admin = new HBaseAdmin(conf)
     if (!admin.isTableAvailable(args(0))) {
-      val tableDesc = new HTableDescriptor(args(0))
+      val tableDesc = new HTableDescriptor(TableName.valueOf(args(0)))
```

Comment:
Sorry, I didn't know when that constructor was added. I understand my proposal makes the compatibility of the earliest version broken.
The other deprication is nothing in the HBase examples, I close this PR.